lib/results_fuzz.go

```diff
@@ -0,0 +1,46 @@
+// +build gofuzz
+
+package vegeta
+
+import (
+	"bytes"
+	"io"
+)
+
+// FuzzResultsFormatDetection tests result list format detection.
+func FuzzResultsFormatDetection(fuzz []byte) int {
+	DecoderFor(bytes.NewReader(fuzz))
+	return 1
+}
+
+// FuzzCSVDecoder tests decoding a CSV format result list.
```

Comment:
We're missing a fuzzer for the default Gob decoder.

Reply:
Added a Gob target.